### ISSUE TITLE
Queued: Duplicate Projects and Stories are permitted

### ISSUE THREAD

**Issue by srivathsar** [CS673Smr16Team1/project#9]
Should we allow adding Projects with the same name?
Should we allow adding Stories with the same name?

**Comment by jaunl** [comment]
Great question. As of right now, we are. But this can be easily fixed if we decide that we should not allow it.

**Comment by sangDev** [comment]
I agree. We can leave this one as an open item for now, as low priority item.

**Comment by srivathsar** [comment]
I think we shouldn't allow duplicate names. Two projects(or story) with the same name is just confusing and useless isn't it? This is just my thought right now :)
Feel free to disagree.

**Comment by srivathsar** [comment]
Being tracked on Bugs.
